client/my-sites/stats/activity-log/index.jsx

```diff
@@ -0,0 +1,153 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { groupBy, map } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteSlug } from 'state/sites/selectors';
+import StatsFirstView from '../stats-first-view';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+import StatsNavigation from '../stats-navigation';
+import ActivityLogDate from '../activity-log-date';
+import QueryActivityLog from 'components/data/query-activity-log';
+import { getActivityLog, isFetchingActivityLog } from 'state/activity-log/selectors';
+import ActivityLogBanner from '../activity-log-banner';
+
+class ActivityLog extends Component {
+
+	componentDidMount() {
+		window.scrollTo( 0, 0 );
+	}
+
+	update_logs( log ) {
+		const { translate } = this.props;
+		switch ( log.type ) {
+			// Scans
+			case 'security_threat_found':
+				log.subTitle = translate( 'Security Threat Found!' );
+				log.icon = 'notice';
+				log.actionText = translate( 'Fix it' );
+			// Backups
+			case 'site_backed_up':
+				log.subTitle = translate( 'Site Backed up' );
+				log.icon = 'checkmark';
+				log.status = 'is-success';
+				log.actionText = 'Restore';
+				break;
+			// Backups
+			case 'site_backed_up_failed':
+				log.subTitle = translate( 'Site Backed up' );
+				log.icon = 'notice';
+				log.actionText = 'Fix';
+				log.status = 'is-error';
+				break;
+			// Plugins
+			case 'plugin_activated':
+				log.subTitle = translate( 'Plugin Activated' );
+				log.icon = 'plugins';
+				log.actionText = 'Deactivate';
+				break;
+			case 'plugin_deactivated':
+				log.subTitle = translate( 'Plugin Deactivated' );
+				log.icon = 'plugins';
+				log.actionText = 'Activate';
+				break;
+			case 'plugin_needs_update':
+				log.subTitle = translate( 'Plugin Update Available' );
+				log.icon = 'plugins';
+				log.status = 'is-warning';
+				log.actionText = 'Update';
+				break;
+			case 'suspicious_code':
+				log.subTitle = translate( 'Security Scan' );
+				log.icon = 'notice';
+				log.status = 'is-error';
+				log.actionText = 'Fix';
+				break;
+			case 'plugin_updated':
+				log.subTitle = translate( 'Plugin Updated' );
+				log.icon = 'plugins';
+				log.actionText = 'Revert';
+				break;
+
+			// Themes
+			case 'theme_switched':
+				log.subTitle = translate( 'Theme Activated' );
+				log.icon = 'themes';
+				log.actionText = 'Revert';
+				break;
+			case 'theme_updated':
+				log.subTitle = translate( 'Theme Updated' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **8** times:
`translate( 'Green Theme Updated' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).